File: 프로그래머스/1/176963. 추억 점수/추억 점수.js
function solution(name, yearning, photo) {
    var answer = Array(photo.length).fill(0);
    for(let i =0; i<photo.length; i++){
        for(let j=0; j<photo[i].length; j++){
            for(let k=0; k<name.length; k++){
                if(photo[i][j] === name[k]){
                    answer[i] += yearning[k];
                }
            }
        }
    }
    return answer;
}

File: 프로그래머스/1/176963. 추억 점수/추억 점수.py
def solution(name, yearning, photo):
    answer = []
    for pic in photo:
        total = 0
        for p in pic:
            for idx,n in enumerate(name):
                if n == p:
                    total += yearning[idx]
                else:
                    total += 0
        answer.append(total)
    return answer